subsys/nrf_security/src/drivers/cracen/cracenpsa/src/kmu.c

```diff
@@ -836,6 +836,13 @@ psa_status_t cracen_kmu_provision(const psa_key_attributes_t *key_attr, int slot
 	case CRACEN_KMU_KEY_USAGE_SCHEME_ENCRYPTED:
 #endif /* PSA_NEED_CRACEN_KMU_ENCRYPTED_KEYS */
 	case CRACEN_KMU_KEY_USAGE_SCHEME_RAW:
+		/* Since AES-keys can be stored using protected there is no reason to allow them to
+		 * be stored raw. This has the added benefit of blocking for export/copy which is
+		 * not supported for AES-keys
+		 */
+		if (psa_get_key_type(key_attr) == PSA_KEY_TYPE_AES) {
```

Comment:
This will also apply for CRACEN_KMU_KEY_USAGE_SCHEME_ENCRYPTED since they share this switch-case code block